fix: Remove background from navigation buttons on Xblock screens

omerhabib26 opened the pull request:
### Description

- Remove navigation button's background

<img src="https://github.com/user-attachments/assets/56b18c34-4d55-4b6a-992f-f4c303c92dc0" width=320 />

Jira Ticket: [LEARNER-10580](https://2u-internal.atlassian.net/browse/LEARNER-10580)

omerhabib26 committed:
fix: Remove background from navigation buttons on Xblock screens

HamzaIsrar12 reviewed (changes requested):
Manual Review: 

| Need Padding | Not Implemented on Videos |
|--------|--------|
| <img width=300 src="https://github.com/user-attachments/assets/552336b6-0b2b-4dc5-86e8-0bdaba6c9f87" />  | <img width=300 src="https://github.com/user-attachments/assets/c11f3c80-3004-40b2-937b-22839b0df552" /> |

omerhabib26 committed:
fix: update spacing issue for webviews and video blocks

- Inject bottom spacing at the bottom of webview using JS
- Add extra spacing at the end of subtitles

HamzaIsrar12 commented on course/src/main/assets/js_injection/webview_bottom_spacer.js:
```diff
@@ -0,0 +1,12 @@
+        const spacer = document.createElement('div');
+        spacer.id = 'bottom-padding';
+        Object.assign(spacer.style, {
+            height: '100px',
```
Please reduce it to `50px`.

HamzaIsrar12 reviewed (changes requested):
Great work overall. 👏🏻 ✨ Minor nit only.

omerhabib26 committed:
fix: Address PR comment
